.devops/templates/build-test-lint.yml

```diff
@@ -0,0 +1,22 @@
+steps:
```

Comment:
## [🕵 FluentUIV7 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=30432&commitId=47ca879c219286f6d254252837a98450ed37734e&env=prod&clientType=FLUENTUI)


  <details><summary><b>Button Action</b> 3 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Button Action.Root - RTL.default.chromium.png|0| Added|
|Button Action.Root - RTL.hover.chromium.png|0| Added|
|Button Action.Root - RTL.pressed.chromium.png|0| Added|

</details>


  <details><summary><b>Fonts</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Fonts.Greek.default.chromium.png|0| Removed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v7"/>